### PR TITLE
endpoint to list all the issues and reworked some of the backend code

### PR DESCRIPTION
**Pull request by iTrip23**

### CODE REVIEW

File: pages/api/content/[id].js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import sql from "~/lib/postgres";
+
+const handler = async ({ query: { id } }, res) => {
+	res.status(200).json({
+		data: await sql`
+	SELECT "content"."title", "content"."description", content.content_type, img_url, video_url FROM "content" INNER JOIN issue_content_rel on "content"."id"= issue_content_rel.content_id INNER JOIN issues on issues.id=${id};`
```
Comment (kerimhudson) [inline pages/api/content/[id].js]: Minor point here but why are you doing content in speech marks for some and not for others? Or just a typo?

File: pages/api/issues/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import sql from "~/lib/postgres";
+
+const handler = async (req, res) => res.status(200).json({ data: await sql`SELECT name, description FROM issues;` });
```
Comment (kerimhudson) [inline pages/api/issues/index.js]: Why are you only returning select information here?

File: pages/api/content/[id].js (hunk 1 above)
Comment (iTrip23) [reply pages/api/content/[id].js]: autocomplete from the valentina studio,
tested the commands in there and copy pasted them.

File: pages/api/issues/index.js (hunk 2 above)
Comment (iTrip23) [reply pages/api/issues/index.js]: I didn't think it was necessary to return all the information. I can modify it and get all the information if that's better

**Review by oksygenn — APPROVED** [review]

**Review by akbasmustafa — APPROVED** [review]
Coding style looks readable and well formatted.

**Pull request merged**